src/kernel/src/fs/mod.rs

```diff
@@ -33,6 +35,7 @@ mod ioctl;
 mod mount;
 mod path;
 mod perm;
+pub mod socket;
```

Comment:
This one should not public otherwise we need to make the other modules here public for consistency.